### ISSUE TITLE
Unable to run the example

### ISSUE THREAD

**Issue by sayjeyhi** [dai-shi/wakuwork#16]
I tried:
```bash
npm create wakuwork
cd wakuwork-example
```

Then installed packages with:
```bash
npm i --legacy-peer-deps
```

And tried to run it:
```bash
npm run dev
```

But got this error:
```bash
➜  wakuwork-example npm run dev             

> wakuwork-example@0.1.0 dev
> NODE_OPTIONS='--loader tsx' wakuwork dev

(node:22229) ExperimentalWarning: --experimental-loader is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
file:///Users/USER/Desktop/projects/RSC/wakuwork-example/node_modules/react-server-dom-webpack/esm/react-server-dom-webpack-node-loader.production.min.js:386
      throw new Error('Expected source to have been transformed to a string.');
            ^

Error: Expected source to have been transformed to a string.
    at transformSource (file:///Users/USER/Desktop/projects/RSC/wakuwork-example/node_modules/react-server-dom-webpack/esm/react-server-dom-webpack-node-loader.production.min.js:386:13)
    at async Loader.moduleStrategy (node:internal/modules/esm/translators:142:17)
    at async link (node:internal/modules/esm/module_job:64:21)
```

Env:
```bash
➜  wakuwork-example node -v                 v16.4.0
➜  wakuwork-example npm -v                  7.18.1
```
M1 mac

**Comment by dai-shi** [comment]
Thanks for reporting!

Hm, it works for me.
```bash
$ node -v
v16.19.0
$ npm -v
8.19.3
```

> Expected source to have been transformed to a string.

I've seen this error and know how to handle it. If I can reproduce this error locally, the fix should be easy...

**Comment by sayjeyhi** [comment]
I was using Volta with Node(`v16.4.0`) so instead, I tried nvm and installed Node `16.19.0` and npm `8.19.3` then it worked.
So I think it was because of nodejs/npm version.

**Comment by dai-shi** [comment]
> So I think it was because of nodejs/npm version.

I'll check it out later.

> It is running but I have lots of failed connections on the console, is it normal?

Yes, it's how vite works.

> BTW one question, can we have SSR with Wakuwork, beside having RSC?

On my todo list, there's opt-in SSR support. But, I don't think it's very easy. Besides, I believe SSR doesn't add much value in our case in my mind. But, we will see. (and to see it, i need to experiment.)

**Comment by dai-shi** [comment]
> I'll check it out later.

node v16.16.0 -> errors
node v16.17.0 -> works

https://github.com/nodejs/node/blob/main/doc/changelogs/CHANGELOG_V16.md?plain=1#L676

So, we depend on a feature available since 16.17.0. I'll add the minimal requirement.
